api/app/responses/api.py

```diff
@@ -332,8 +345,10 @@ def get(self, event_id):
                 'tags': [_serialize_tag(rt.tag, language) for rt in response.response_tags]
             }
 
+            if (event.event_type == EventType.CONTINUOUS_JOURNAL or event.event_type == EventType.JOURNAL):
```

Comment:
Think this condition isn't needed. We might want to use action-editors for other event types as well :)